.env

```diff
@@ -0,0 +1 @@
+REACT_APP_JWT_SECRET=maazamart
```

Comment:
don't push env file on GitHub

Reply:
Thanks for suggestion. Will correct it.👍